trillian/ctfe/instance.go

```diff
@@ -172,14 +172,20 @@ func SetUpInstance(ctx context.Context, client trillian.TrillianLogClient, cfg *
 }
 
 // ValidateLogMultiConfig checks that a config is valid for use with multiple
-// backend log servers. The backend set must define a set of log backends
-// with distinct (non empty) names and non empty backend specs. The log configs
-// must all specify a log backend and each must be one of those defined in
-// the backend set. The prefixes of configured logs must all be distinct and
-// must not be empty.
+// backend log servers. The rules applied are:
+//
+// 1. The backend set must define a set of log backends with distinct
+// (non empty) names and non empty backend specs.
+// 2. The log configs must all specify a log backend and each must be one of
+// those defined in the backend set.
+// 3. The prefixes of configured logs must all be distinct and must not be
+// empty.
+// 4. The set of tree ids for each configured backend must be distinct.
+// 5. The backend specs must all be distinct.
```

Comment:
(nit) 5. could move up and become 2. so the rules for backends are together

Reply:
Done.